src/components/dateTimePicker/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {fireEvent, render} from '@testing-library/react-native';
+import DateTimePicker, {DateTimePickerProps} from '../index';
+import {act} from 'react-test-renderer';
+
+const testID = 'dateTimePicker';
+const TestCase = (props: Partial<Omit<DateTimePickerProps, 'dialogProps'>>) => {
+  const defaultProps: DateTimePickerProps = {
+    value: new Date('2021-04-04T00:00:00Z'),
+    migrateDialog: true,
+    testID
+  };
+  return <DateTimePicker {...defaultProps} {...props}/>;
+};
+
+const someDate = new Date('2021-04-04T00:00:00Z');
+const someDateNextDay = new Date(someDate.getTime() + 24 * 60 * 60 * 1000);
+const someDateNextHour = new Date(someDate.getTime() + 60 * 60 * 1000);
+jest.mock('../../../optionalDependencies', () => {
```

Comment:
Not sure. jest-setup will mock this for every test file we have. I only see external modules in the jest-mock file i think its more organized this way.
Adding it to the jest-setup file also works. I guess we could do that. Do you think it will be clearer to put it there?

Reply:
Yes, I think that's the proper place for it because if we'll have an issue in private we'll look there first for a mock.

Reply:
What kind of issue will possibly have? When I you the jest mock here I don't think it has any effect on other test files. If I put it in the jest-setup one it does.

Reply:
If we'll need to mock the `DateTimePickerPackage` we'll probably look there and not here